kernel/kernel-websubclient-api/src/main/java/io/mosip/kernel/websub/api/config/publisher/RestTemplateHelper.java

```diff
@@ -0,0 +1,26 @@
+package io.mosip.kernel.websub.api.config.publisher;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+
+@Component
+public class RestTemplateHelper {
+
+	@Value("${mosip.auth.filter_disable:true}")
```

Comment:
Which auth filter is this and why it is by default disabled ?

Reply:
@vishwa-vyom  auth filter of websub. there is a same property in websub conf we have to make both false to run authz and authn